hardhat.config.ts

```diff
@@ -302,6 +308,14 @@ const config: HardhatUserConfig = {
           browserURL: "https://optimistic.etherscan.io/",
         },
       },
+      {
+        network: "basesepolia",
+        chainId: 84532,
+        urls: {
+          apiURL: "https://api-sepolia.basescan.org/api",
+          browserURL: "https://sepolia.basescan.org/",
+        },
+      },
```

Comment:
We also need to add base sepolia api key [here](https://github.com/VenusProtocol/venus-protocol/pull/529/files#diff-57cf5378f8cab20b02b279097ba6fad08eb53f747d81770045dda9c5f95effe3R228)

Reply:
https://github.com/VenusProtocol/venus-protocol/pull/529/commits/03b21e4997bdd6a0e9ec08f749699c111bbc7183